src/Twig/Runtime/TextRuntime.php

```diff
@@ -40,7 +41,7 @@ public function __construct(LoggerInterface $systemLogger, Slugify $slugify)
      */
     public function jsonDecode($string)
     {
-        return json_decode($string, true);
+        return Json::parse($string);
```

Comment:
There is an invisible `try/catch` here, it just isn't showing up in your IDE … yet :stuck_out_tongue_winking_eye: 

Oh, and as this will come up,  the native [`json_decode`](http://php.net/manual/en/function.json-decode.php) returns a `null` if decoding fails.